runtime/peaq-dev/src/lib.rs

```diff
@@ -485,7 +485,7 @@ impl pallet_collective::Config<CouncilCollective> for Runtime {
 parameter_types! {
 	pub const ProposalBond: Permill = Permill::from_percent(5);
 	pub const ProposalBondMinimum: Balance = DOLLARS;
-	pub const SpendPeriod: BlockNumber = DAYS;
+	pub const SpendPeriod: BlockNumber = DAYS/96;
```

Comment:
We have the below constant; please use this because it's more straightforward.

```
pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
pub const HOURS: BlockNumber = MINUTES * 60;
```

Reply:
This constants may be used in other places too. In my view, changing this may have side effects.

Reply:
I mean, we should use MINUTES * 15 to represent 15 min instead of using DAYS / 96.

Reply:
I committed the required modification please